homeassistant/components/rainmachine/__init__.py

```diff
@@ -361,57 +270,41 @@
     return True
 
 
+async def async_migrate_entry(hass, config_entry):
```

Comment:
If we can, we should avoid migrations. It prevents users from rolling back to an older version. Since this data is not impacting the new runtime, we're probably better off not doing a migration and keeping the data around.